codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/RustSettings.kt

```diff
@@ -28,14 +28,25 @@ private const val RUNTIME_CONFIG = "runtimeConfig"
 private const val CODEGEN_SETTINGS = "codegen"
 private const val LICENSE = "license"
 
-data class CodegenConfig(val renameExceptions: Boolean = true, val includeFluentClient: Boolean = true, val formatTimeoutSeconds: Int = 20) {
+data class CodegenConfig(
+    // / Rename `Exception` to `Error` in the generated SDK
```

Comment:
Something strange happened to the doc comments

Reply:
woah. serves me right for using the Rust format I guess